README.md

```diff
@@ -1,407 +1,436 @@
-```
-
-Search over variable references from your Language Server.
-
-```vim
-nnoremap <Leader>en <cmd>lua require'telescope.builtin'.find_files{ cwd = "~/.config/nvim/" }<CR>
-```
-
-Find all the files in your nvim config.
 
-### Available keys for `defaults`
+## Customization
 
-- `generic_sorter`:
-    - Description: The sorter to be used for generic searches.
-    - `default`: `require('telescope.sorters').get_generic_fuzzy_sorter
-- `file_sorter`:
-    - Description: The sorter to be used for file based searches.
-    - `default`: `require('telescope.sorters').get_fuzzy_file
+You can configure `telescope.nvim` at two levels:
+  globally, through the `setup()` method,
+  or in a finer-grained manner
+  by passing options to individual built-in pickers.
 
-### Full Example
+|  [Presentation](#presentation) | [Sorting](#sorting) | [Mappings](#mappings)
 
-```vim
-lua <<EOF
--- totally optional to use setup
-require('telescope').setup {
-  defaults = {
-    shorten_path = false -- currently the default value is true
-  }
-}
-EOF
+#### Presentation
 
-nnoremap <c-p> :lua require'telescope.builtin'.find_files{}<CR>
-nnoremap <silent> gr <cmd>lua require'telescope.builtin'.lsp_references{ shorten_path = true }<CR>
-```
+| Description                                           | Keys                   | Options                    |
+|-------------------------------------------------------|------------------------|----------------------------|
+| Where the prompt should be located.                   | `prompt_position`      | top/bottom                 |
+| What should the prompt prefix be.                     | `prompt_prefix`        | string                     |
+| Where first selection should be located.              | `sorting_strategy`     | descending/ascending       |
+| How the telescope is drawn.                           | `layout_strategy`      | center/horizontal/vertical |
+| How transparent is the telescope window should be.    | `winblend`             | NUM                        |
+| Layout specific configuration ........ TODO           | `layout_defaults`      | TODO                       |
+| TODO                                                  | `width`                | NUM                        |
+| TODO                                                  | `preview_cutoff`       | NUM                        |
+| TODO                                                  | `results_height`       | NUM                        |
+| TODO                                                  | `results_width`        | NUM                        |
+| The border chars, it gives border telescope window    | `borderchars`          | dict                       | 
+| Whether to color devicons or not                      | `color_devicons`       | boolean                    |
+| Whether to use less or bat .. TODO                    | `use_less`             | boolean                    |
```

Comment:
not only `bat` but also `cat` if `bat` is not installed

Reply:
What do mean? the option is `use_less`, does it mean it uses cat instead of less? or there is another option I'm not aware of @Conni2461

Reply:
I mean it should be something like: `Whether to use less of cat/bat` because if bat is not installed cat is default

Reply:
Ohh I see what you mean @Conni2461 . I think `Whether to use less of cat/bat` sounds good. Done